torch/_inductor/fx_passes/post_grad.py

```diff
@@ -663,14 +663,16 @@ def decompose_auto_functionalized(graph):
     def replacement(match: Match, *args, **kwargs):
         from torch._higher_order_ops.auto_functionalize import auto_functionalized_dense
 
+        only_clone_tensors = tuple(match.nodes[0].meta.get("only_clone_tensors", []))
```

Comment:
do we expect this always to be set ?

Reply:
It may not be set if the reinplacer decided that nothing can be reinplaced.